pkg/utils/graph/eventhandler_seed.go

```diff
@@ -132,6 +133,15 @@ func (g *graph) handleSeedCreateOrUpdate(seed *gardencorev1beta1.Seed) {
 		}
 	}
 
+	for _, defaultDNS := range seed.Spec.DNS.Defaults {
+		// TODO(dimityrmirchev): handle/add support for workload identities
+		if defaultDNS.CredentialsRef.APIVersion == corev1.SchemeGroupVersion.String() &&
+			defaultDNS.CredentialsRef.Kind == "Secret" {
+			secretVertex := g.getOrCreateVertex(VertexTypeSecret, defaultDNS.CredentialsRef.Namespace, defaultDNS.CredentialsRef.Name)
+			g.addEdge(secretVertex, seedVertex)
```

Comment:
Out of curiosity, how were added the vertex and the edge before making the default domains explicit configuration in the seed spec?